ports/raspberrypi/boards/pimoroni_inky_frame_5_7/pins.c

```diff
@@ -39,6 +71,7 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_INKY_RES), MP_ROM_PTR(&pin_GPIO27) },
     { MP_ROM_QSTR(MP_QSTR_INKY_DC), MP_ROM_PTR(&pin_GPIO28) },
 
+    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_CYW0) },
```

Comment:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_PICO_LED), MP_ROM_PTR(&pin_CYW0) },
```

I'd rather the default be visible on the front.

Reply:
Ok, fine. This keeps the pico-led.